src/path_util.tl

```diff
@@ -0,0 +1,31 @@
+
```

Comment:
this project depends on [cyan](github.com/teal-language/cyan), which has a decent [path module](https://teal-language.github.io/cyan/index.html#cyan.fs.path) that can do these operations